voltdbclient/authentication_test.go

```diff
@@ -23,21 +23,6 @@ import (
 	"testing"
 )
 
-func TestLoginRequest(t *testing.T) {
```

Comment:
I'm sorry, I answered this in a hurry. Yes this is tested in the `wire` subpackage